pkg/controller/dataimportcron-controller.go

```diff
@@ -263,28 +263,28 @@ func (r *DataImportCronReconciler) update(ctx context.Context, dataImportCron *c
 
 	desiredDigest := dataImportCron.Annotations[AnnSourceDesiredDigest]
 	digestUpdated := desiredDigest != "" && (len(imports) == 0 || desiredDigest != imports[0].Digest)
+	incrementMetric := true
```

Comment:
Be positive and use a name like `upToDate` instead here and in the func args

Reply:
Why not straight up say if inc/dec is the desired action?